src/main/resources/static/js/main.js

```diff
@@ -1,76 +1,78 @@
-let mainBlock = document.getElementById('main')
-let authBlock = document.getElementById('auth')
-let infoBlock = document.getElementById('plyer-info')
-let lobbyBlock = document.getElementById('lobby')
-let startGameButton = document.getElementById('start-game-button')
-let playersList = document.getElementById('players-list');
+let mainBlock = $('#main')
+let authBlock = $('#auth')
+let infoBlock = $('#player-info')
+let lobbyBlock = $('#lobby')
+let startGameButton = $('#start-game-button')
+let playersList = $('#players-list')
+
+let contentTypeHeader = {'Content-Type': 'application/json; charset=UTF-8'}
 
 renderMainWindow()
 
 function auth() {
-    const xhttp = new XMLHttpRequest();
-    let playerName = document.getElementsByName('regName')[0].value
+    let playerName = $('#regName')[0].value
     if (playerName === undefined || playerName === '') {
         console.log("Empty name")
         return
     }
-    xhttp.open("POST", "api/authorization", true);
-    xhttp.setRequestHeader('Content-type', 'application/json; charset=UTF-8')
-    xhttp.onload = function() {
-        renderMainWindow()
-    }
-    xhttp.withCredentials = true
-    let postObj = {
-        name: playerName
-    }
-    xhttp.send(JSON.stringify(postObj));
+    $.post({
+        url: 'api/authorization',
+        headers: contentTypeHeader,
+        dataType: 'json',
+        data: JSON.stringify({name: playerName}),
+        success: function () {
+            renderMainWindow()
+        },
```

Comment:
Can't we do the following? This function wrapper looks like it doesn't make any sense
```suggestion
        success: renderMainWindow()
```

Reply:
If you put it this way, "Sign in" button only works on the second click. I don't why. BTW static analyzer in IntelliJ IDEA highlights your variant.
![image](https://user-images.githubusercontent.com/70219831/201494274-6ac01210-f9ec-4693-8c40-7853ea9a99d2.png)

Reply:
Oh, well, now I see it, `success` must be some kind of callback declaration, not the call itself.